apps/console/src/features/identity-providers/api/identity-provider.ts

```diff
@@ -734,6 +735,49 @@ export const getLocalAuthenticators = (): Promise<LocalAuthenticatorInterface[]>
         });
 };
 
+/**
+ * Get Local Authenticator details from `t/<TENANT>>/api/server/v1/configs/authenticators/<AUTHENTICATOR_ID>`
+ *
+ * @param {string} id - Authenticator ID.
+ * @return {Promise<AuthenticatorInterface>} Response as a promise.
+ * @throws {IdentityAppsApiException}
+ */
+export const getLocalAuthenticator = (id: string): Promise<AuthenticatorInterface> => {
+
+    const requestConfig = {
+        headers: {
+            "Accept": "application/json",
+            "Access-Control-Allow-Origin": store.getState().config.deployment.clientHost,
```

Comment:
```suggestion
```
I think we do not need this line as it is the server that should be setting this header attribute. We need to do this change in other places as well.

Reply:
+1
Created an issue to remove from other places.
https://github.com/wso2/product-is/issues/12589